src/styles/main.scss

```diff
@@ -27,8 +27,9 @@ body {
 }
 
 .products__container {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $productWidth);
```

Comment:
everything is great in the code you've created, just if you use a certain number in  grid-template-columns property, the pictures will always stay in the place where they are, not moving to the next row when the screen width gets more narrow.
https://prnt.sc/thp11y

this is how it looks like when I'm trying to make the width of the screen smaller.